packages/contracts/contracts/voting/addresslist/AddresslistVoting.sol

```diff
@@ -212,7 +207,15 @@ contract AddresslistVoting is MajorityVotingBase {
     /// @inheritdoc MajorityVotingBase
     function _canVote(uint256 _proposalId, address _voter) internal view override returns (bool) {
         Proposal storage proposal_ = proposals[_proposalId];
-        return _isVoteOpen(proposal_) && isListed(_voter, proposal_.snapshotBlock);
+
+        if (
+            proposal_.configuration.voteMode != VoteMode.VoteReplacement &&
+            proposal_.voters[_voter] != VoteOption.None
+        ) {
+            revert VoteReplacementNotAllowed();
```

Comment:
Shouldn't we simply return `false`?

Reply:
Yes, good catch. Fixed in ffdd16247d90ffb83c9c5addbeacc957858abc95